package/MDAnalysis/auxiliary/XVG.py

```diff
@@ -64,17 +96,32 @@ def uncomment(lines):
         # if comment_position == 0, then the line is empty
 
 class XVGStep(base.AuxStep):
-    """ AuxStep class for .xvg file format 
+    """ AuxStep class for .xvg file format.
+
+    Extends the base AuxStep class to allow selection of time and 
+    data-of-interest fields (by column index) from the full set of data read 
+    each step.
 
     Parameters
     ----------
-    time_selector : int
+    time_selector : int | None, optional

```

Comment:
Note that sphinx will concatenate the docstring of `__init__` at the end 
of the class docstring. Whenever the option you choose, do not write a 
full fledge docstring in init (do not add a description), have there 
only the parameters and returns.


On 08-08-16 14:38, Max Linke wrote:
>
> In package/MDAnalysis/auxiliary/XVG.py 
> <https://github.com/MDAnalysis/mdanalysis/pull/868#discussion_r73867547>:
>
> >
> >      Parameters
> >      ----------
> > -    time_selector : int
> > +    time_selector : int | None, optional
>
> You are free ti choose where it should be documented. We do it in the 
> init function.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/MDAnalysis/mdanalysis/pull/868/files/92ec8fb2bbc3ca6dcb67b8cbee59637ff23a5194..48599f38392051f506435e8fa9933697f6c85902#r73867547>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ABUWul9lDtuitJ7iA88Yr5S5slparJNDks5qdyNFgaJpZM4It8C_>.
>